src/main/java/frc/robot/commands/shooter/RampThenShoot.java

```diff
@@ -26,14 +37,24 @@ public RampThenShoot(
     this(intake, indexer, shooter, limelight, null, null);
   }
 
+  /**
+   * Constructs a RampThenShoot command that rumbles the controllers if the Limelight doesn't hub
```

Comment:
reword? "rumbles the controllers if the Limelight doesn't hub isn't visible" is weird